source/refined-github.css

```diff
@@ -114,3 +114,22 @@ pr-branches
 .js-issue-row .text-small.text-gray {
 	line-height: 1.8;
 }
+
+/* Removes redundant user/repo from notifications when being grouped by repository */
+/* Place discussion number before the title */
+.js-notifications-group [id^='notification'] {
+	flex-direction: row !important;
+}
+/* Hide duplicate repository URL */
+.js-notifications-group [id^='notification'] > .d-flex > .f6 {
+	font-size: 0 !important;
+}
+/* Size and position the discussion number */
+.js-notifications-group [id^='notification'] > .d-flex > .f6 > span {
+	display: block;
+	font-size: 12px;
+	font-variant-numeric: tabular-nums;
```

Comment:
Added to be careful about some fonts with non-tabular numbers.

@fregante WDYT? Leave this here or remove it?